README.md

```diff
@@ -1,34 +1,35 @@
-# Packetvisor 3.0
+# Packetvisor 3.0 (Rust)
+This is ported in Rust from C
```

Comment:
Don't need to mention it is rust project

Reply:
The suggestion is reflected